rpcs3/rpcs3qt/debugger_frame.h

```diff
@@ -36,6 +36,30 @@ enum class system_state : u32;
 class instruction_editor_dialog;
 class register_editor_dialog;
 
+struct ThreadComboBox : public QComboBox
+{
+	Q_OBJECT
+
+public:
+	ThreadComboBox(QWidget* parent) : QComboBox(parent) {}
+	virtual ~ThreadComboBox() {}
+
+
+	void mousePressEventComboBox(QMouseEvent* e)
+	{
+		QComboBox::mousePressEvent(e);
+	}
+
+public Q_SLOTS:
+	virtual void mousePressEvent(QMouseEvent* e) override
+	{
+		UpdateUnitList(std::make_shared<QMouseEvent>(e->type(), e->localPos(), e->windowPos(), e->screenPos(), e->button(), e->buttons(), e->modifiers(), e->source()));
+	}
+
+Q_SIGNALS:
+	void UpdateUnitList(std::shared_ptr<QMouseEvent>);
+};
```

Comment:
I have no idea what you're trying to achieve here.

Reply:
It hooks the mouse press event of the combo box

Reply:
Why

Reply:
I reverted this because it caused some bug